IDE/STM32Cube/wolfssl_example.c

```diff
@@ -1468,25 +1469,264 @@ static void PrintMemStats(void)
 #endif
 }
 
-#if 0
-static void* wolfMallocCb(size_t size)
+/* UART DMA IO Routines */
```

Comment:
Please add a build option around the TLS test code to enable a way to disable it for folks just looking to do the wolfCrypt test/benchmark. These changes won't work for smaller micros.

You could gate on `!defined(WOLFCRYPT_ONLY) && defined(WOLFSSL_TLS13) && !defined(NO_WOLFSSL_SERVER) && !defined(NO_WOLFSSL_CLIENT)`

Reply:
I like your gating idea.  Will be fixed in next commit.